### DISCUSSION TITLE
Organização de tarefas via Trello

### DISCUSSION THREAD

**Discussion by swxtz** [diego3g/rsxp-2023#47] [General]
Eu pensei em organizar as funções no Trello, seria mas fácil de organizar assim

**Comment by leoferreiralima** [comment]
Opa @swxtz acredito que organizar aqui no github com issues, milestones e um project facilite pois está tudo no mesmo lugar e vamos conseguir de uma maneira facil ver o andamento do projeto

**Comment by diego3g** [comment]
Isso, vou criar um project board que funciona muito semelhante ao Trello. 

Vou fechar essa discussão já que já tem bastante discussão aberta e to quase me perdendo aqui 😂
